src/Atmos/Model/turbulence.jl

```diff
@@ -89,7 +89,7 @@ $(DocStringExtensions.FIELDS)
 """
 struct ConstantViscosityWithDivergence{FT} <: TurbulenceClosure
   "Dynamic Viscosity [kg/m/s]"
-  ρν::FT
+  ρν::SVector{3,FT}
```

Comment:
Why is this needed?

Reply:
This was done for testing purposes, so we could set different constant viscosities in different directions.

Reply:
Since I've already done tests with this, I can remove this and revert the changes for the constant viscosity closure, would that be alright @jkozdon ?

Reply:
I would suggest reverting it then if its not needed in the final PR